open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.CONFIG

```diff
@@ -1,17 +1,34 @@
 [*
 **  PowerOn Name:       BANNO.NEWSUBCREATE.V1.CONFIG
 **  Letterfile Name:    BANNO.NEWSUBCREATE.V1.CFG
```

Comment:
I suggest changing the comment on line 3 "Letterfile Name:" to "Datafile Name:"

Reply:
Makes sense - will do